extra-resources/mods/Mythos/common.cardset

```diff
@@ -118,9 +145,38 @@ card('Healer') {
         }
     }
 }
-
+// https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg
+// CC BY-SA 3.0
+// https://commons.wikimedia.org/wiki/User:SteveMcCluskey
```

Comment:
I suggest going through the images and changing the attributions.

For each file, transform the current link, such as `https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg` to the `commons.wikimedia.org` domain, such as `https://commons.wikimedia.org/wiki/File:Ricci_Guangqi_2.jpg`, and click on "Use this file on the web"

So this:

    // https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg
    // CC BY-SA 3.0
    // https://commons.wikimedia.org/wiki/User:SteveMcCluskey

Should become this:

    // https://commons.wikimedia.org/wiki/File:Ricci_Guangqi_2.jpg
    // By Kircher, Athanasius, 1602-1680. [CC BY-SA 3.0 (http://creativecommons.org/licenses/by-sa/3.0)], via Wikimedia Commons